### PR TITLE
move project type to the AppType

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit review please
Moved `project_type` to `AppType`

File: unicoremc/models.py [hunk 1]
```diff
@@ -63,17 +63,26 @@ class Meta:
...
+    SPRINGBOARD = 'springboard'
+    PROJECT_TYPES = (
+        (UNICORE_CMS, 'unicore-cms'),
+        (SPRINGBOARD, 'springboard'),
+    )
+
     name = models.CharField(max_length=256, blank=True, null=True)
     title = models.TextField(blank=True, null=True)
+    project_type = models.CharField(
+        choices=PROJECT_TYPES, max_length=256, default=UNICORE_CMS)
```
Comment (Rizziepit) [inline unicoremc/models.py:76]: `default=SPRINGBOARD` instead? But keeping it so that the migration stores `UNICORE_CMS` for existing apps

**Comment by Rizziepit** [comment]
+1 aside from one comment

File: unicoremc/models.py (hunk 1 above)
Comment (miltontony) [inline unicoremc/models.py:76]: Yeah, I'm keeping it for the sake of migrations. We can change it later on.

**Pull request merged**